### ISSUE TITLE
fontSize prop does not seem to work

### ISSUE THREAD

**Issue by zandernelson** [conorhastings/react-native-syntax-highlighter#5]
When I set the fontSize prop the font is not affected, but the margin between the lines seems to be affected.

```
export default class App extends React.Component {
  render() {
    return (
      <View>
        <SyntaxHighlighter 
        language='javascript'
        fontFamily='Courier'
        fontSize={45}
        style={docco}
        >
        {"function(num){\n\tconsole.log(num)\n}"}
        </SyntaxHighlighter>
      </View>
    );
  }
}
```
Result. Notice large space between the lines. I messed around with the src code and adjusting the default parameter value for the fontSize seemed to change the size. Not sure what the fix is though, new to React.

![image1](https://user-images.githubusercontent.com/11861331/28801913-d545e678-7621-11e7-866b-bd13861728d2.PNG)

**Comment by conorhastings** [comment]
@zander312 thanks for the issue! I'll try to take a look this weekend

**Comment by conorhastings** [comment]
fixed in `v1.2.3` , thanks for the issue!
